apps/web/src/composables/environment.ts

```diff
@@ -8,7 +8,10 @@ const origin = window.location.origin
 
 /* Ethereum environment */
 const ethereumUrl = import.meta.env.PUBLIC_ETHEREUM_RPC_URL || "http://127.0.0.1:8545"
+const ethereumWebSocketUrl = import.meta.env.PUBLIC_ETHEREUM_WEBSOCKET_URL
```

Comment:
@ccali11 do you mind find and replacing:

ETHEREUM_WEBSOCKET_URL => ETHEREUM_WS_URL
ethereumWebSocketUrl => ethereumWsUrl
webSocketProvider => wsProvider

Reply:
For the whole wide project.